ebuild-writing/users-and-groups/text.xml

```diff
@@ -5,85 +5,140 @@
 
 <body>
 <p>
-If your ebuild requires a user or group to be added for a daemon, for example,
-this should be performed via the functions available in <c>user.eclass</c>.
-Regardless of whether you are adding a group or a user, this should be performed
-in the <c>pkg_setup</c> phase function. <c>pkg_setup</c> is called
-before the compile process, with
-<uri link="::general-concepts/sandbox">sandbox</uri>
-disabled, so a build that requires the user to exist will
-have it, and is also called for both binary and source packages. You may also
-use the <c>pkg_preinst</c> or <c>pkg_postinst</c> functions for user creation, if
-the user is not required during or before <c>src_install</c>.
+The modern way of creating users and groups is governed by <uri
+link="https://gentoo.org/glep/glep-0081.html">GLEP 81</uri>.  It is implemented
+via <c>acct-user</c> and <c>acct-group</c> eclasses.
+</p>
+
+<p>
+New users and groups are created as packages respectively in <c>acct-user</c>
+and <c>acct-group</c> categories.  First, you need to choose UIDs/GIDs that
+are not used by any other packages.  You should consult the <uri
+link="https://wiki.gentoo.org/wiki/Project:Quality_Assurance/UID_GID_Assignment">
+UID/GID assignment list</uri> maintained by the QA project, and possibly
+registries maintained by other distributions that are linked there.  Afterwards,
```

Comment:
```suggestion
the UID/GID registries maintained by other distributions that are linked on the same page.  Afterwards,
```

It is not linked "in the list". As a side note, the wiki refers to them as "databases" rather than "registries". It doesn't make a difference to me, just wanted to point it out.